orchagent/muxorch.cpp

```diff
@@ -795,22 +805,28 @@ bool MuxNbrHandler::enable(bool update_rt)
         IpPrefix pfx = it->first.to_string();
         if (update_rt)
         {
-            if (remove_route(pfx) != SAI_STATUS_SUCCESS)
-            {
-                return false;
-            }
-            updateTunnelRoute(nh_key, false);
+            route_ctx_list.push_back(MuxRouteBulkContext(pfx));
         }
 
+        updateTunnelRoute(nh_key, false);
+
         it++;
     }
 
+    if (update_rt && !removeRoutes(route_ctx_list))
+    {
+        return false;
+    }
+
+    gRouteBulker.clear();
```

Comment:
should we clear this even if its returned false above

Reply:
Took a look at this, it's not needed since we call clear from removeRoutes